specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/FailoverManagedInstance.json

```diff
@@ -15,44 +15,24 @@
   "produces": [
     "application/json"
   ],
-  "security": [
-    {
-      "azure_auth": [
-        "user_impersonation"
-      ]
-    }
-  ],
-  "securityDefinitions": {
-    "azure_auth": {
-      "type": "oauth2",
-      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
-      "flow": "implicit",
-      "description": "Azure Active Directory OAuth2 Flow",
-      "scopes": {
-        "user_impersonation": "impersonate your user account"
-      }
-    }
-  },
   "paths": {
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/failover": {
       "post": {
         "tags": [
-          "ManagedInstances"
+          "FailoverManagedInstance"
         ],
         "description": "Failovers a managed instance.",
-        "operationId": "ManagedInstances_Failover",
+        "operationId": "FailoverManagedInstance_Failover",
         "parameters": [
           {
-            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
           },
           {
-            "$ref": "#/parameters/SubscriptionIdParameter"
-          },
-          {
-            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
-          },
-          {
-            "$ref": "#/parameters/ManagedInstanceNameParameter"
+            "name": "managedInstanceName",
```

Comment:
@MilanBrkicFON 
this is due to your doc override the name parameters in: https://msdata.visualstudio.com/Database%20Systems/_git/DsMainDev?path=%2FSql%2Fxdb%2Fmanifest%2Fsvc%2Fmgmt%2FWebApiHosting%2FARM%2FV20190601%2FManagedInstance%2FControllers%2FFailoverManagedInstance.cs&version=GBmaster&line=66&lineEnd=68&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

if removing the doc override, it will generate "ManagedInstanceNameParameter". Thus, this is not a true breaking change.